cvat/requirements/base.txt

```diff
@@ -22,3 +22,4 @@ scipy==1.0.1
 sqlparse==0.2.4
 django-sendfile==0.3.11
 dj-pagination==2.3.2
+pascal-voc-writer==0.1.4
```

Comment:
What do you think if we have separate requirements.txt file for scripts directory? Does it make sense?

Reply:
I guess it depends on the amount of scripts that CVAT will have and how does the requirements grow for each one, if there will be already two scripts (coco and pascal) and each one has their own requirements, it may be a good idea to split them from the base. Should I move the requirement to a new `cvat/requirements/scripts.txt`?

Reply:
@syonekura ,

Current internal branch has the following structure:
```
cvat/utils/
    convert_to_coco.py
    convert_to_mask.py
    doc/
        convert_to_coco.md
        convert_to_mask.md
    README.md
    requirements.txt
```